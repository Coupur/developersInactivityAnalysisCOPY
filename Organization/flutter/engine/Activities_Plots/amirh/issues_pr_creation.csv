id,date,creator_login
199089064,2017-01-05 23:44:47,amirh
266319587,2017-10-18 00:11:29,amirh
266560208,2017-10-18 16:35:58,amirh
266672826,2017-10-18 23:19:25,amirh
267238626,2017-10-20 16:53:57,amirh
267265572,2017-10-20 18:34:16,amirh
267303300,2017-10-20 20:58:16,amirh
270166273,2017-11-01 01:26:29,amirh
270823129,2017-11-02 22:06:54,amirh
271069807,2017-11-03 18:11:26,amirh
271148028,2017-11-03 23:54:37,amirh
271994959,2017-11-07 21:44:57,amirh
272404893,2017-11-09 01:02:02,amirh
272697285,2017-11-09 19:42:36,amirh
273045994,2017-11-10 20:13:18,amirh
273947246,2017-11-14 21:21:46,amirh
273971052,2017-11-14 22:46:05,amirh
274367206,2017-11-16 01:48:39,amirh
274713528,2017-11-17 00:38:23,amirh
274954561,2017-11-17 18:27:25,amirh
275022467,2017-11-17 23:03:26,amirh
275042068,2017-11-18 01:32:50,amirh
275882085,2017-11-21 22:30:34,amirh
276230779,2017-11-22 23:22:12,amirh
283964343,2017-12-21 18:18:34,amirh
284040783,2017-12-22 00:19:13,amirh
286395377,2018-01-05 20:24:28,amirh
286959347,2018-01-09 03:25:41,amirh
287920891,2018-01-11 20:54:50,amirh
288687836,2018-01-15 18:55:30,amirh
289436026,2018-01-17 22:25:50,amirh
290595620,2018-01-22 19:58:42,amirh
292603649,2018-01-29 23:49:53,amirh
337741109,2018-07-03 04:24:56,amirh
338732749,2018-07-05 21:32:09,amirh
340437673,2018-07-11 23:17:00,amirh
340722769,2018-07-12 17:05:34,amirh
342479729,2018-07-18 20:28:00,amirh
344103189,2018-07-24 16:02:20,amirh
344570357,2018-07-25 19:09:48,amirh
345387042,2018-07-27 22:45:46,amirh
347138751,2018-08-02 19:12:09,amirh
347200352,2018-08-02 22:45:21,amirh
347564767,2018-08-03 23:48:01,amirh
348516715,2018-08-07 23:01:59,amirh
349627302,2018-08-10 19:12:30,amirh
351759786,2018-08-17 22:44:12,amirh
352241584,2018-08-20 18:13:44,amirh
352323918,2018-08-20 22:51:28,amirh
353869309,2018-08-24 17:37:42,amirh
354861133,2018-08-28 18:59:53,amirh
362725441,2018-09-21 18:21:06,amirh
362780747,2018-09-21 21:30:43,amirh
362782483,2018-09-21 21:38:02,amirh
369291786,2018-10-11 20:27:26,amirh
371177615,2018-10-17 17:03:55,amirh
371746099,2018-10-18 23:02:37,amirh
372057066,2018-10-19 17:57:05,amirh
374218815,2018-10-26 04:06:05,amirh
374573692,2018-10-26 22:57:47,amirh
375128572,2018-10-29 17:22:17,amirh
376094312,2018-10-31 18:07:00,amirh
376136846,2018-10-31 19:59:13,amirh
376185381,2018-10-31 22:31:44,amirh
376587520,2018-11-01 21:58:39,amirh
376657019,2018-11-02 03:33:07,amirh
376960914,2018-11-02 20:42:43,amirh
377614012,2018-11-05 22:40:35,amirh
377632981,2018-11-05 23:51:57,amirh
377936768,2018-11-06 16:47:03,amirh
377996089,2018-11-06 19:20:18,amirh
378029449,2018-11-06 20:55:38,amirh
378416621,2018-11-07 18:36:37,amirh
378881903,2018-11-08 19:27:35,amirh
378907689,2018-11-08 20:40:34,amirh
378935444,2018-11-08 22:01:30,amirh
378947952,2018-11-08 22:41:51,amirh
378971252,2018-11-09 00:13:24,amirh
379247315,2018-11-09 17:11:16,amirh
379971676,2018-11-12 22:08:28,amirh
384823723,2018-11-27 14:35:03,amirh
394247845,2018-12-26 21:56:00,amirh
394457120,2018-12-27 18:10:24,amirh
394487700,2018-12-27 21:04:02,amirh
394512028,2018-12-27 23:49:20,amirh
410628860,2019-02-15 06:48:06,amirh
417505789,2019-03-05 21:18:34,amirh
422500898,2019-03-19 01:19:38,amirh
423426351,2019-03-20 19:10:38,amirh
423571288,2019-03-21 04:48:41,amirh
423970612,2019-03-21 22:32:29,amirh
426211937,2019-03-27 22:30:17,amirh
427498556,2019-04-01 03:46:12,amirh
446265875,2019-05-20 19:08:16,amirh
452162307,2019-06-04 19:14:30,amirh
452715900,2019-06-05 21:02:41,amirh
453111233,2019-06-06 15:54:58,amirh
454335659,2019-06-10 19:35:38,amirh
457755553,2019-06-19 00:44:06,amirh
459348738,2019-06-21 19:50:41,amirh
459373579,2019-06-21 21:08:34,amirh
460520925,2019-06-25 16:22:19,amirh
462179537,2019-06-28 19:35:23,amirh
468308869,2019-07-15 20:00:22,amirh
468818702,2019-07-16 19:12:58,amirh
