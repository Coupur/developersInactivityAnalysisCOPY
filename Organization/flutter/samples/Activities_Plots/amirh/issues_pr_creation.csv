id,date,creator_login
389393436,2018-12-10 16:55:37,amirh
