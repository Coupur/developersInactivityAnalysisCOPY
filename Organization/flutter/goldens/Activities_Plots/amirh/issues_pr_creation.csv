id,date,creator_login
346745222,2018-08-01 19:55:40,amirh
